### PR TITLE
Feature/vaana 295 initialise client frontend 4 v2

### PR DESCRIPTION
**Pull request by AbdoulayeNDAO121**
PR for theming app

### CODE REVIEW

File: src/app/app.component.css [hunk 1]
```diff
@@ -0,0 +1,19 @@
+.buttonEl {
```
Comment (morlombakop) [inline src/app/app.component.css:1]: @AbdoulayeNDAO121  where are these buttons?

File: src/custom-theme.scss [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  300: #f39200,
+  400: #f39200,
+  500: #f39200,
+  contrast: (
+    500: white,
+    A200: white,
+  ),
+);
+
+$frontend-client-primary: mat-palette($mat-primary);
```
Comment (morlombakop) [inline src/custom-theme.scss:40]: let call it `vaanah` instead of `front-client` what do you think?

Comment (AbdoulayeNDAO121) [reply src/custom-theme.scss:40]: Ok for me

File: src/app/app.component.css (hunk 1 above)
Comment (AbdoulayeNDAO121) [reply src/app/app.component.css:1]: No i didn't delete the app.component.html file, it is

Comment (AbdoulayeNDAO121) [reply src/app/app.component.css:1]: image

Comment (morlombakop) [reply src/app/app.component.css:1]: > No i didn't delete the app.component.html file

Yes bad it is there. Add the screenshot on the description

File: src/app/app.module.ts [hunk 3]
```diff
@@ -2,13 +2,14 @@ import { NgModule } from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
 import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
 import { TranslateModule } from '@ngx-translate/core';
+import { SharedModule } from './shared/shared.module';
```
Comment (morlombakop) [inline src/app/app.module.ts:5]: This should be at line 7 or 9 with others. Let's not mix imports

**Review by morlombakop — APPROVED** [review]

**Pull request merged**